### PR TITLE
Add  `recruitment/list-candidates` profile `breezy-hr` map

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds `breezy-hr` map for `recruitment/list-candidates` profile.

Known limitation of the map: Breezy HR provider does provide paginated endpoint for listing candidates, but the response do not contain previous and next page offsets or page links. Map implementation do allow the client to pass page limit and page cursor, but do not provide page cursors in the perform outcome.

The PR is based on #328.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: grid/recruitment/list-candidates/maps/breezy-hr.suma [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  }
+  
+  http GET "/v3/company/{ parameters.COMPANY_ID }/position/{ input.jobId }/candidates" {
+    request {
+      query {
+        page_size = input.limit
+        page = input.page
+      }
+      headers {
+        Authorization = parameters.ACCESS_TOKEN
```
Comment (zdne) [inline grid/recruitment/list-candidates/maps/breezy-hr.suma:27]: Would none of the provider security work for this? (E.g. ApiKeyHeaderSecurity)

File: grid/recruitment/list-candidates/maps/breezy-hr.suma [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        Authorization = args.accessToken
+      }
+    }
+
+    response 200 {       
+      return {}
+    }
+
+    response 500 "application/json" {
+      // Breezy HR returns http status code 500 in case that the job ID doesn't exist. It could be caused by other reasons as well, but at least the error helps the client understand the error better.
```
Comment (zdne) [inline grid/recruitment/list-candidates/maps/breezy-hr.suma:165]: 😭

**Review by zdne — APPROVED** [review]

File: grid/recruitment/list-candidates/maps/breezy-hr.suma (hunk 1 above)
Comment (janhalama) [reply grid/recruitment/list-candidates/maps/breezy-hr.suma:27]: You are right ApiKeyHeaderSecurity security will work. It should be used here.

Comment (janhalama) [reply grid/recruitment/list-candidates/maps/breezy-hr.suma:27]: I will update the provider definition and related maps in separate PR.

**Pull request merged**